core-library/src/main/java/org/silverpeas/core/contribution/attachment/webdav/impl/WebdavContentDescriptor.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2000 - 2020 Silverpeas
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * As a special exception to the terms and conditions of version 3.0 of
+ * the GPL, you may redistribute this Program in connection with Free/Libre
+ * Open Source Software ("FLOSS") applications as described in Silverpeas's
+ * FLOSS exception.  You should have received a copy of the text describing
+ * the FLOSS exception, and it is also available here:
+ * "https://www.silverpeas.org/legal/floss_exception.html"
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.silverpeas.core.contribution.attachment.webdav.impl;
+
+import org.silverpeas.core.contribution.attachment.model.SimpleDocument;
+
+import java.time.OffsetDateTime;
+
+/**
+ * This class represents the description of a webdav content.
+ * @author silveryocha
+ */
+public class WebdavContentDescriptor {
```

Comment:
WebDAV is an extension to the HTTP protocol. Can this class to be renamed RemoteContentDescriptor (or a name like this) meaning it is a descriptor about a content accessible/editable remotely whatever the protocol used here?
By this way, it can refers both the content of a document accessed through WebDAV, WOPI, and later CMIS